.circleci/config.yml

```
@@ -729,6 +730,13 @@ jobs:
             if [ "$MIGRATE_FLAG" == "true" ]; then
               NODE_ENV=production ./scripts/reindex/index-docket-entries/index-docket-entries.ts 5
             fi
+      - run:
+          name: Populate test users in Cognito
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "test" ]; then
+                  export REGION="us-east-1"
+                  ./scripts/user/setup-glued-users.ts
+            fi
```

We only run glue jobs to test currently, but we've run glue jobs to environments other than test before.

```suggestion
            export REGION="us-east-1"
            ./scripts/user/setup-glued-users.ts
```